appium/webdriver/webdriver.py

```diff
@@ -337,7 +337,8 @@ def find_element(self, by: str = AppiumBy.ID, value: Union[str, Dict] = None) ->
             `appium.webdriver.webelement.WebElement`: The found element
 
         """
-        # TODO: If we need, we should enable below converter for Web context
+        # Let's keep it for future support of the selenium workaround
```

Comment:
I'd rather change the comment to `We prefer to patch locators in the client code. Checking current context every time a locator is accessed could significantly slow down tests`